src/sqlancer/sqlite3/SQLite3Provider.java

```diff
@@ -49,12 +55,26 @@ public class SQLite3Provider extends SQLProviderAdapter<SQLite3GlobalState, SQLi
     public static boolean allowFloatingPointFp = true;
     public static boolean mustKnowResult;
 
+    private Map<String, String> queryPlanPool;
+
+    // Data structures for QPG
```

Comment:
Come to think of it: Could we integrate some of the code here in the `DatabaseProvider` class? I assume some variables and code here are common to all `DatabaseProvider` implementations?